dino-ml/scanner.py

```diff
@@ -1,13 +1,16 @@
 from PIL import Image
+from PIL import ImageGrab
 from datetime import datetime
 import os
 
-dino_color = (83, 83, 83, 255)
+dino_color = (83, 83, 83)
 
 def screenshot(x, y, w, h):
-    os.system("screencapture -R{},{},{},{} tmp.png".format(x, y, w, h))
-    img = Image.open("tmp.png")
-    return img
+    # os.system("screencapture -R{},{},{},{} tmp.png".format(x, y, w, h))
```

Comment:
it can be done same as the main code( taking screenshots of custom window size) by the following code : 

import win32gui

...


def screenshot(x, y, w, h):
    
    toplist, winlist = [] ,[]
    def enum_cb(hwnd, results):
        winlist.append((hwnd, win32gui.GetWindowText(hwnd)))
    win32gui.EnumWindows(enum_cb, toplist)

    chrome = [(hwnd, title) for hwnd, title in winlist if 'chrome' in title.lower()]
    # just grab the hwnd for first window matching chrome
    chrome = chrome[0]
    hwnd = chrome[0]
    
    win32gui.SetForegroundWindow(hwnd)
    bbox = win32gui.GetWindowRect(hwnd)
    img = ImageGrab.grab(bbox=(x,y,w,h))
    img.save('tmp.PNG')
    return img